btx/diagnostics/run.py

```diff
@@ -35,6 +35,19 @@ def __init__(self, exp, run, det_type):
         self.rank = self.comm.Get_rank()
         self.size = self.comm.Get_size() 
 
+    def init_base_powders(self, img_shape):
+        """
+        Initialize the base powders: sum, sum of squares, max, and min.
+
+        Parameters
+        ----------
+        img_shape : tuple, 3d
+            The shape of unassembled, calibrated images: (n_panels, n_x, n_y)
+        """
+        self.powders = dict()
+        for key in ['sum', 'sqr', 'max', 'min']:
+            self.powders[key] = np.zeros(img_shape)
+
```

Comment:
This is not needed, please remove as it is redundant with `compute_base_powders()`

Reply:
An error occurs.